end_users/README.md

```diff
@@ -1,3 +1,24 @@
 # End Users
 
-Documentation coming soon...
+## Table of Contents
+
+* [Merge Options](#merge-options)
+  * [Setting the Correct Merge Option](#setting-the-correct-merge-option)
+
+## Merge Options
+
+Repositories that use BonnyCI should not use any merge strategy which changes the SHA of the commit. This is because [Zuul](https://github.com/openstack-infra/zuul), the scheduler used in BonnyCI, will be thrown off by the change in SHA when it merges commits that successfully pass the gate queue.
```

Comment:
@eggshell: Maybe we should add another section in the lore that explains the parts of BonnyCI? Then this section could link to it.

@jesusaurus: Would you mind providing a suggested revision to help make this clear?